crates/sui-indexer/src/models/checkpoints.rs

```diff
@@ -21,6 +23,8 @@ pub struct Checkpoint {
     pub transactions: Vec<Option<String>>,
     pub previous_checkpoint_digest: Option<String>,
     pub end_of_epoch: bool,
+    #[diesel(sql_type = Text)]
```

Comment:
why do you need this, would it work if you remove this one?

Reply:
We do not.